### PR TITLE
Collect full db dump

### PR DESCRIPTION
**Pull request by didib**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [] Is the subject and message clear and concise?
- [] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

File: sos/plugins/__init__.py [hunk 1]
```diff
@@ -569,7 +569,7 @@ def add_copy_spec(self, copyspecs, sizelimit=None, tailit=True):
         a single file the file will be tailed to meet sizelimit. If the first
         file in a glob is too large it will be tailed to meet the sizelimit.
         """
-        sizelimit = sizelimit or self.get_option("log_size")
+        sizelimit = sizelimit if sizelimit is not None else self.get_option("log_size")
```
Comment (bmr-cymru) [inline sos/plugins/__init__.py]: This isn't going to work as a single line, as it exceeds the pep8 length limit. We can either stuff `get_option("log_size")` into a local, or rework it as a simple branched `if: ... else:`.

File: sos/plugins/__init__.py [hunk 2]
```diff
@@ -703,7 +703,7 @@ def add_cmd_output(self, cmds, suggest_filename=None,
             cmds = [cmds]
         if len(cmds) > 1 and (suggest_filename or root_symlink):
             self._log_warn("ambiguous filename or symlink for command list")
-        sizelimit = sizelimit or self.get_option("log_size")
+        sizelimit = sizelimit if sizelimit is not None else self.get_option("log_size")
```
Comment (bmr-cymru) [inline sos/plugins/__init__.py]: Ditto.

**Comment by didib** [comment]
I don't like the review process on github. Not sure how to mark that I added Signed-Off-By.

**Comment by didib** [comment]
Not sure how to mark that I added Signed-Off-By.

**Comment by didib** [comment]
Sorry for spamming - when added the second comment, I couldn't see the first one, so thought there might have been some problem submitting it... Did I say I do not like github? I almost always find gerrit much cleaner.

**Comment by didib** [comment]
Now verified that it works. Created a dummy huge table, ran old sosreport and saw that the dump is partial, patched with current PR, ran again and saw that it's complete.

**Comment by pmoravec** [comment]
ACK from me, codechange is sane, verified on my system.

**Pull request closed without merging**